integration-tests/src/test/java/oracle/kubernetes/operator/ItOperatorUpgrade.java

```diff
@@ -293,6 +229,20 @@ private void checkDomainRollingRestarted() throws Exception {
     }
   }
 
+  /**
+   * Check whether the weblogic server instances are still RUNNING 
+   * not restarted due to Operator Upgrade
+   *
+   * @throws Exception If restarted
+   */
+  private void checkDomainNotRestarted() throws Exception {
+    TestUtils.checkPodReady(DUID + "-" + domain.getAdminServerName(), DOM_NS);
+    for (int i = 2; i >= 1; i--) {
```

Comment:
if rolling restart is triggered, do u see it immediately or will there be little lag? checking once immediately is enough to make sure rolling restart is not triggered?

Reply:
Combining here since @markxnelson made the same comment below... Checking that the operator doesn't restart pods is a bit tricky.  The operator will respond very quickly to check all existing domains after it starts, so if the operator hasn't started the deletion of any pods within a few seconds then it isn't rolling the servers.  You can check if pods are in the process of being deleted by checking the .metadata.deletionTimestamp.

Reply:
Here should I expect a NULL value for {.metadata.deletionTimestamp}
$ kubectl get po/operatordomain250-managed-server1 -n weblogic-domain250 -o  jsonpath={.metadata.deletionTimestamp}

Reply:
Yes, that's right.  Oh, and for completeness you should check that the .metadata.creationTimesamp.  I don't think it's an or...

Reply:
I added the logic to check the creationTimesamp before and after upgrade to make sure that the server pod has not been re-stared due to upgrade.